### DISCUSSION TITLE
Test de la sirène d’extérieur Zigbee Neo NAS-AB06B2 - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#32] [Comments]
# Test de la sirène d’extérieur Zigbee Neo NAS-AB06B2 - Haade.fr

Test de la sirène solaire zigbee tuya Neo NAS-AB06B2 conçue pour l’extérieur, dans home assistant à moins de 40€. Pour ce prix vaut-elle le coût ?

https://haade.fr/fr/blog/test-alarme-exterieur-batterie-solaire-tuya-neo-NAS-AB06B2-zigbee

**Comment by Razor648** [comment]
Pourtant dans les devices pris en charge par Z2MQTT, je vois bien le paramètre "battery" :
https://www.zigbee2mqtt.io/devices/NAS-AB06B2.html
Tu confirmes que cela fonctionne à présent ? Sinon très bon test, merci à toi !

**Comment by tirso58** [comment]
Je peux ajouter quelques commentaites suite à un modèle que j'ai installé.
1 - Je confirme, l'état de la baterie ne remonte pas.
2 - J'ai un soucis de perte de visibilité (le last seen peut remonter à 11h donc inutilisable car non fiable), malgré qu'elle soit à quelques centimetres d'un routeur.
3 - Oui, non pris en charge par ZHA
4 - Impossible de faire une mise à niveau?

Bref, pas convaincu en l'état.

**Reply by haade-administrator** [reply]
non j'ai essayé de l'intégrer à Z2M mais impossible de faire remonter les données
